ERR_MODULE_NOT_FOUND during pnpm run dev

**Describe the bug**
I have tried to launch the E2E tests following the contribution page.
Bit when ever i launch `pnpm run dev`, it displays this error.

```
[1] /home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:366
[1]     throw new ERR_MODULE_NOT_FOUND(
[1]           ^
[1] CustomError: Cannot find module '/home/valls/workspace/web/mermaid.git/.vite/server' imported from /home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/dist-raw/runmain-hack.js
[1]     at finalizeResolution (/home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:366:11)
[1]     at moduleResolve (/home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:801:10)
[1]     at Object.defaultResolve (/home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:912:11)
[1]     at /home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/src/esm.ts:218:35
[1]     at entrypointFallback (/home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/src/esm.ts:168:34)
[1]     at /home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/src/esm.ts:217:14
[1]     at addShortCircuitFlag (/home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/src/esm.ts:409:21)
[1]     at resolve (/home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/src/esm.ts:197:12)
[1]     at resolve (/home/valls/workspace/web/mermaid.git/node_modules/.pnpm/ts-node@10.9.1_jrs6fgrkrfl5zdawlcdiuhuotq/node_modules/ts-node/src/child/child-loader.ts:15:39)
[1]     at nextResolve (node:internal/modules/esm/loader:163:28)
[1] ts-node-esm .vite/server exited with code 1
```
Any idea if it could explain that i can't execute successfully the E2E tests?

**Desktop (please complete the following information):**

- OS: Debian 10

**Additional context**
Add any other context about the problem here.

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
     "build:types": "concurrently \"tsc -p ./packages/mermaid/tsconfig.json --emitDeclarationOnly\" \"tsc -p ./packages/mermaid-mindmap/tsconfig.json --emitDeclarationOnly\"",
     "build:watch": "pnpm build:vite --watch",
     "build": "pnpm run -r clean && concurrently \"pnpm build:vite\" \"pnpm build:types\"",
-    "dev": "concurrently \"pnpm build:vite --watch\" \"ts-node-esm .vite/server\"",
+    "dev": "concurrently \"pnpm build:vite --watch\" \"ts-node-esm .vite/server.ts\"",
     "docs:build": "ts-node-esm --transpileOnly packages/mermaid/src/docs.mts",
     "docs:verify": "pnpm docs:build --verify",
     "todo-postbuild": "documentation build src/mermaidAPI.ts src/config.ts src/defaultConfig.ts --shallow -f md --markdown-toc false > src/docs/Setup.md && prettier --write src/docs/Setup.md",

